### PR TITLE
Help Page completed

### PR DESCRIPTION
**Pull request by inkerton**
image

### CODE REVIEW

**Comment by inkerton** [comment]
This is in response to the issue #98

**Review by anuj123upadhyay — APPROVED** [review]

File: src/main.jsx [hunk 1]
```diff
@@ -112,13 +115,19 @@ const router = createBrowserRouter([
             path: "/termsandconditions",
             element: <TermsAndConditions />,
             },
+
+            {
+            path: "/helpPage",
+            element: <HelpPage />,
+
```
Comment (anuj123upadhyay) [inline src/main.jsx:123]: {
            path: "/helpPage",
            element: <HelpPage />


here curly bracket is not closed so this causes issue in deployment , hey @inkerton  please rectify it and raise pr

Comment (anuj123upadhyay) [reply src/main.jsx:123]: in line 122  of  src/main.jsx

**Review by anuj123upadhyay — COMMENTED** [review]
{
path: "/helpPage",
element:

here curly bracket is not closed so this causes issue in deployment , hey @inkerton please rectify it and raise pr

in line 122 of src/main.jsx

File: src/main.jsx (hunk 1 above)
Comment (inkerton) [reply src/main.jsx:123]: ohkay my mistake I'll do it right now.

**Comment by inkerton** [comment]
@anuj123upadhyay Is this ok now you should be able to merge.

**Comment by anuj123upadhyay** [comment]
hey @inkerton  you have this error in your code in deployment 
ScreenshotVercel
Resolve this error then make a new fork and pr as till time i merge some others pr also

**Comment by anuj123upadhyay** [comment]
> hey @inkerton you have this error in your code in deployment ScreenshotVercel Resolve this error then make a new fork and pr as till time i merge some others pr also

Add } instead of )

**Comment by inkerton** [comment]
is it okay now? @anuj123upadhyay

**Comment by inkerton** [comment]
Merge this one too please @anuj123upadhyay

**Comment by inkerton** [comment]
@anuj123upadhyay Please add these as soon as Possible I will work forward then .

**Pull request merged**